packages/slider/src/presenters/SliderPresenter.js

```diff
@@ -13,7 +15,31 @@ export default class SliderPresenter extends Component {
     max: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     min: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
     step: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+    variant: PropTypes.oneOf(AVAILABLE_SLIDER_TYPES)
+  };
+
+  getTickMarks = (min, max, step, styles) => {
+    const loops = step ? Math.floor((max - min) / step) : 1;
+    const tickMarks = [];
+    for (let i = 0; i < loops; i += 1) {
+      tickMarks.push(
+        <div
+          className={css(styles.markRules)}
+          style={{ left: `${(i * 100) / loops}%` }}
+          key={`mark-${i}`}
+        />
+      );
+    }
+    // adds the mark to end of slider
+    tickMarks.push(
+      <div
+        className={css(styles.markRules)}
+        style={{ left: "100%" }}
+        key={`mark-${loops + 1}`}
```

Comment:
Setting `left: 100%` actually places the last tick mark 1px after the end. I have attached an image.

Reply:
OK, I see. I've changed the style to fix it.